### ISSUE TITLE
Add a random passphrase generator (feature request)

### ISSUE THREAD

**Issue by ryancdotorg** [keybase/warpwallet#16]
It would be helpful to include a random passphrase generator in warpwallet that produces passphrases with at least an 80 bit security level on their own, and encourage it's use.

**Comment by lirazsiri** [comment]
Good idea to do that in a way that still makes makes it easy for the user to add additional entropy on his own.

5 diceware words = 65 bits of entropy
6 diceware words = 78 bits of entropy

So a passphrase with at least 5 diceware words + user supplied entropy is going to be good even if the CSPRNG malfunctions.
